spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/bedrock/anthropic3/BedrockAnthropic3ChatProperties.java

```diff
@@ -44,6 +44,11 @@ public class BedrockAnthropic3ChatProperties {
 	 */
 	private String model = AnthropicChatModel.CLAUDE_V3_SONNET.id();
 
+	/**
+	 * Set model timeout. 1000 ms by default.
+	 */
+	private Long timeout = 1000L;
```

Comment:
- Use Duration as a type. 
- Don't create new property file for each model.  Instead add the timeout property to the common [BedrockAwsConnectionProperties](https://github.com/spring-projects/spring-ai/blob/main/spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/bedrock/BedrockAwsConnectionProperties.java)